src/components/DataDrawer/Chart/index.tsx

```diff
@@ -11,77 +12,118 @@ type ChartProps = {
   config: ChartConfig;
 };
 
-export function Chart({ title, data, config }: ChartProps) {
+function getChartConfig(stacked: boolean, title: string) {
+  return {
+    title: {
+      fontColor: '#CCC',
+      display: true,
+      text: title,
+      fontSize: 20,
+    },
+    scales: {
+      xAxes: [
+        {
+          stacked,
+          gridLines: {
+            display: false,
+          },
+          ticks: {
+            fontColor: '#CCC',
+          },
+        },
+      ],
+      yAxes: [
+        {
+          ticks: {
+            fontColor: '#CCC',
+          },
+          stacked,
+          gridLines: {
+            display: false,
+          },
+        },
+      ],
+    },
+    legend: {
+      display: false,
+      position: 'right',
+    },
+  } as ChartOptions;
+}
+
+function formatChartData(data: TableData, config: ChartConfig) {
+  const transpose = config.transpose || false;
   const header = data.rows[0];
   const tableRows = data.rows.slice(1, data.rows.length);
+
   const colors = shuffle(
     colormap({
       colormap: 'rainbow-soft',
-      nshades: tableRows.length,
+      nshades: !transpose ? tableRows.length : header.length,
```

Comment:
Maybe we can get rid of the extra `!`. (One of Ben's wise comments)
`transpose ? header.length : tableRows.length`

Reply:
I used it on purpose to make sure that we read the code in a "normal" way first. But I could be convinced otherwise...